source/d_geoblocks.rst

```diff
@@ -0,0 +1,151 @@
+=========
+Geoblocks
+=========
+
+Introduction
+============
+
+Geo-information services often rely on high resolution data from multiple sources.
+Geoblocks provide the tool to process these geo-data on-the-fly. 
+The technology enables big geo-data solutions for operational data flows.
+Geoblock operations can be computed on a regular basis or executed upon request.
+
+One geoblock can modify the information in raster or geometry data, e.g. multiply two rasters or buffer a geometry.
+Multiple geoblocks can be stacked together into a model, like stacking lego blocks into a tower.
+
+Geoblocks information services are transparent and reproducible.
+A geoblocks graph is the model configuration that defines how data should be combined to create information.
+The model configuration can be accessed and altered via API and a graph view visualization is automatically generated (example in figure x).
+
+https://demo.lizard.net/api/v4/geoblocks/e12fcc5c-7313-827b-1d5c-26bd294792c4/visualize/?format=svg
+
+Figure x: Geoblocks visualization example of a water depth raster masked below 2.5 cm.
+
+.. warning::
+    Missing Figure!
+    Linked geoblock is no longer available
+
+Flexibility
+===========
+
+- Geoblocks are modular. This means that the blocks are self-contained units, that can be combined or interchanged with others to create different shapes or designs. Hence, multiple geoblocks configurations can be combined into a new configuration or information service.
```

Comment:
Hier heb je een voorbeeld van een plek waar je twee zinnen in één bullet point nodig hebt.
Dat is de reden dat ik altijd hoofdletters en punten gebruik.